roles/openstack/provision/network/defaults/main.yml

```diff
@@ -0,0 +1,17 @@
+# Network defaults
+
+external_network: 180
+timeout_network: 180
+shared_network: False
+
+# Subnet defaults
+
+enable_dhcp: True
+dns_servers:
+    - 8.8.8.7
+    - 8.8.8.8
```

Comment:
cool. I actually wasn't familiar of it. 
so what is exactly the advantage here? I mean why opendns > google dns